src/Fetch.ml

```diff
@@ -379,6 +381,48 @@ module Response = struct
   external clone : t = "clone" [@@bs.send.pipe: t]
 end
 
+module FormData = struct 
+  module EntryValue = struct
+    type t
+
+    let classify t =
+      if Js.typeof t = "string" then `String (Obj.magic t)
+      else `File (Obj.magic t)
+  end
+
+  module Iterator = Fetch__Iterator
+  type t = formData
+
+  external make : unit -> t = "FormData" [@@bs.new]
+  external append : name:string -> value:string -> unit = "append" [@@bs.send.pipe : t]
+  external delete : string -> unit = "delete" [@@bs.send.pipe : t]
+  external get : string -> EntryValue.t option = "get" [@@bs.send.pipe : t]
+  external getAll : string -> EntryValue.t array = "getAll" [@@bs.send.pipe : t]
+  external set : name:string -> value:string -> unit = "set" [@@bs.send.pipe : t]
+  external has : string -> bool = "has" [@@bs.send.pipe : t]
+  external keys : t -> string Iterator.t = "keys" [@@bs.send]
+  external values : t -> EntryValue.t Iterator.t = "values" [@@bs.send]
+
+  external appendObject : name:string -> value:< .. > Js.t -> ?filename:string -> unit =
+    "append" [@@bs.send.pipe : t]
+
+  external appendBlob : name:string -> value:blob -> ?filename:string -> unit =
+    "append" [@@bs.send.pipe : t]
+
+  external appendFile : name:string -> value:file -> ?filename:string -> unit =
+    "append" [@@bs.send.pipe : t]
+
+  external setObject : name:string -> value:< .. > Js.t -> ?filename:string -> unit =
+    "set" [@@bs.send.pipe : t]
+
+  external setBlob : name:string -> value:blob -> ?filename:string -> unit =
+    "set" [@@bs.send.pipe : t]
+
+  external setFile : name:string -> value:file -> ?filename:string -> unit =
+    "set" [@@bs.send.pipe : t]
```

Comment:
These labels seem unnecessary to me. First because it seems unlikely that anyone's going to confuse the order of `name` and `value`, as this is pretty obvious and consistent over any API I've ever seen in any language. Secondly, since this binds to an existing API, there's both precedent for and an expectation for it to be unlabelled to model it as closely as possible.